packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -313,6 +300,30 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
 	): Promise<{ snapshotTree: ISnapshotTree; sequenceNumber: number }>;
 }
 
+/**
+ * @experimental
+ * @deprecated - These APIs are unstable, and can be changed at will. They should only be used with direct agreement with the Fluid Framework.
+ * @legacy
+ * @alpha
+ * @sealed
+ */
+export interface StageControlsExperimental {
```

Comment:
Since these are tagged and are also otherwise inaccessible from non-experimental APIs, maybe just go ahead with naming `StageControls`

Reply:
i prefer to keep it verbose for now. we can re-evaluate before we actually merge this outside the test branch